package-lock.json

```diff
@@ -8116,6 +8116,7 @@
             "color-convert": "^1.9.0"
           }
         },
+<<<<<<< HEAD
```

Comment:
this is what Travis is choking on.

besides fixing it manually, you're also welcome to just delete the file and rerun `npm install` in the root of the repo to recreate it.